crates/lang/macro/tests/compile_tests.rs

```diff
@@ -53,4 +54,6 @@ fn compile_tests() {
     t.compile_fail("tests/ui/fail/S-04-non-storage-ink-impls.rs");
     t.compile_fail("tests/ui/fail/S-05-storage-as-event.rs");
     t.compile_fail("tests/ui/fail/S-06-event-as-storage.rs");
+
+    t.pass("tests/ui/chain_extension/simple.rs");
```

Comment:
Why not name it consistent to the other tests a la `E-01-simple-chain-extension.rs`?

Reply:
tbh i dislike the current naming scheme but we cannot do much better with current trybuild.